ddtrace/llmobs/_integrations/openai.py

```diff
@@ -161,10 +161,11 @@ def _llmobs_set_meta_tags_from_completion(
         if isinstance(prompt, str):
             prompt = [prompt]
         span.set_tag_str(INPUT_MESSAGES, json.dumps([{"content": str(p)} for p in prompt]))
-        parameters = {"temperature": kwargs.get("temperature", 0)}
-        if kwargs.get("max_tokens"):
-            parameters["max_tokens"] = kwargs.get("max_tokens")
+
+        parameters = {k: v for k, v in kwargs.items() if k not in ("model", "prompt")}
+        parameters["temperature"] = kwargs.get("temperature", 0)
```

Comment:
Is this line here ensuring _temperature_ always has a default value? I think the default value of temperature for completions is `1` ((source)[https://platform.openai.com/docs/api-reference/chat/create?lang=python])

IMO we should avoid hard coding the logic that temperature defaults to 0. Similar to the other arguments, we should only set temperature if it's able to be extracted out of kwargs.